simulations.go

```diff
@@ -0,0 +1,304 @@
+// Code generated by the Paddle SDK Generator; DO NOT EDIT.
+
+package paddle
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+
+	"github.com/PaddleHQ/paddle-go-sdk/pkg/paddlenotification"
+)
+
+// Simulation: Represents a simulation entity.
+type Simulation struct {
+	// ID: Unique Paddle ID for this simulation, prefixed with `ntfsim_`.
+	ID string `json:"id,omitempty"`
+	// Status: Whether this entity can be used in Paddle.
+	Status Status `json:"status,omitempty"`
+	// NotificationSettingID: Paddle ID of the notification setting where this simulation is sent, prefixed with `ntfset_`.
+	NotificationSettingID string `json:"notification_setting_id,omitempty"`
+	// Name: Name of this simulation.
+	Name string `json:"name,omitempty"`
+	// Type: Single event sent for this simulation, in the format `entity.event_type`.
+	Type SimulationTypeName `json:"type,omitempty"`
+	// Payload: Simulation payload. `null` for scenarios.
+	Payload paddlenotification.NotificationPayload `json:"payload,omitempty"`
```

Comment:
I think this does need to be a pointer, as the API can return `null` for scenarios (the comment above indicates this too)

Reply:
Added an example in https://github.com/PaddleHQ/paddle-go-sdk/pull/30/commits/fdb71c857d962f6d0bc51e716193215be83cbcca to demonstrate this can be `nil`